src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,30 @@
-export const UserInfo = () => {};
+import React from 'react';
+
+import usersFromServer from '../../api/users';
+
+import { User } from '../../react-app-env';
+
+interface Props {
+  userId: number;
+}
+
+export const UserInfo: React.FC<Props> = ({
+  userId,
+}) => {
+  let user: User | undefined = usersFromServer.find(u => u.id === +userId);
```

Comment:
```suggestion
  const user: User | undefined = usersFromServer.find(u => u.id === userId);
```
userId already number. looks like no need to make number from number.